GalaxyZooTouchTable/GalaxyZooTouchTable/ClassificationPanel.xaml.cs

```diff
@@ -12,14 +16,29 @@ public partial class ClassificationPanel : UserControl
         public ClassificationPanel(UserConsole parent)
         {
             InitializeComponent();
-            this.Console = parent;
+            Console = parent;
         }
 
         private void CloseButton_TouchUp(object sender, System.Windows.Input.TouchEventArgs e)
         {
-            this.Console.StartButton.Visibility = System.Windows.Visibility.Visible;
-            this.Console.ButtonArc.Visibility = System.Windows.Visibility.Visible;
-            this.Console.ControlPanel.Children.Remove(this);
+            MoveClassifier();
+            Console.MoveButton();
+            Console.ClassifierOpen = !Console.ClassifierOpen;
+        }
+
+        public void MoveClassifier()
+        {
+            float StartPos = Console.ClassifierOpen ? -600 : 0;
+            float EndPos = Console.ClassifierOpen ? 400 : -600;
+
+            var PanelTransform = new TranslateTransform(0, StartPos);
+            RenderTransform = PanelTransform;
+            DoubleAnimation panelAnimation = new DoubleAnimation(EndPos, new Duration(TimeSpan.FromSeconds(0.25)));
+            if (Console.ClassifierOpen)
+            {
+                panelAnimation.Completed += (s, e) => Console.ControlPanel.Children.Remove(this);
```

Comment:
I know your not using them, but could you use more descriptive names for `s` and `e`